packages/design-system/src/components/card/MyBookmarkCard.tsx

```diff
@@ -0,0 +1,58 @@
+import { Icon } from '@/icons';
+import chippiNoImage from '../../assets/chippi_no_image.svg';
+import BaseCard from './BaseCard';
+
+interface MyBookmarkCardProps {
+  title: string;
+  content?: string;
+  category: string;
+  imageUrl?: string;
+  date?: string;
+}
```

Comment:
```suggestion
}type BaseProps = {
  title: string;
  content?: string;
  category: string;
  imageUrl?: string;
};

type RemindProps = BaseProps & {
  type: 'remind';
  timeRemaining: string; // 필수
  date?: never;         // 금지
};

type BookmarkProps = BaseProps & {
  type: 'bookmark';
  date: string;         // 필수
  timeRemaining?: never; // 금지
};

type CardProps = RemindProps | BookmarkProps;
```
타입 분리해서 사용하는거 좋을거같아요-!! 다만 이런식으로 타입별 필수,불가 필드를 분리해서 사용하는 방식은 어떤가요?